tests/modules/picard/markduplicates/test.yml

```diff
@@ -12,4 +12,4 @@
   tags:
     - picard
     - picard/markduplicates
-  exit_code: 1
+
```

Comment:
Add a path and an md5sum here if possible?

Reply:
Good catch